PartyPlaylistsTests/UnitTests/RoomSongDataStoreUnitTests.cs

```diff
@@ -0,0 +1,68 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using PartyPlaylists.Contexts;
+using PartyPlaylists.Models.DataModels;
+using PartyPlaylists.Services;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace PartyPlaylistsTests.UnitTests
+{
+    [TestClass]
+    public class RoomSongDataStoreUnitTests
+    {
+        private PlaylistContext playlistContext;
```

Comment:
Rename context with underscore
```suggestion
        private PlaylistContext _playlistContext;
```